### PR TITLE
Simplify code a bit

### PR DESCRIPTION
**Pull request by nyurik**
This is a partial simplification from the other PR, without any breaking changes.

See also #71

### CODE REVIEW

File: src/sprite/serialize.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@ use resvg::usvg::Rect;
 use serde::ser::SerializeSeq;
 use serde::{Serialize, Serializer};
 
-/// Custom Serde field serialiser for [`Rect`].
+/// Custom Serde field serializer for [`Rect`].
```
Comment (flother) [inline src/sprite/serialize.rs:5]:
```suggestion
/// Custom Serde field serialiser for [`Rect`].
```

I _think_ I'm at the fifth stage of grief and I've accepted that I need to write code in US English, but prose is a different matter. Serde's `Serializer` is a type of serialiser, and so the One True English (aka British English) should be used here ;)

File: src/sprite/serialize.rs [hunk 2]
```diff
@@ -31,7 +31,7 @@ where
     }
 }
 
-/// Custom Serde field serialiser for a vector of [`Rect`]s.
+/// Custom Serde field serializer for a vector of [`Rect`]s.
```
Comment (flother) [inline src/sprite/serialize.rs:34]:
```suggestion
/// Custom Serde field serialiser for a vector of [`Rect`]s.
```

File: src/sprite/serialize.rs [hunk 3]
```diff
@@ -60,7 +60,7 @@ where
...
```
Comment (flother) [inline src/sprite/serialize.rs:63]:
```suggestion
/// Custom Serde field serialiser for a vector of [`Rect`]s.
```

**Review by flother — CHANGES_REQUESTED** [review]
I have just a handful of pedantic quibbles over language, but otherwise ready to merge

File: src/sprite/serialize.rs (hunk 1 above)
Comment (nyurik) [reply src/sprite/serialize.rs:5]: lol, i didn't even realize the british eng spelled it differently. TIL

**Comment by nyurik** [comment]
Fixed, let me know if anything else is needed. Ready to merge :)

**Review by flother — APPROVED** [review]
Thanks!

**Pull request merged**